src/hooks/usePantry.ts

```diff
@@ -181,9 +181,52 @@ function entry({ project }: { project: string }): Entry {
     const versions = dir.join("versions.txt")
     return { dir, yml, versions }
   }
-  throw new TeaError('not-found: pantry: package.yml', {project})
-}
+  //check for fuzzy string
+  const closestPkg = getClosestPackageSuggestion(project, prefix);
```

Comment:
Probably simpler to just `await` this line.

Reply:
I tried using await here, but for it to the work the `entry` function must be wrapped in async but it gives some error. I will try it out though.